### PR TITLE
DOCS-3541 Service Catalog (beta) docs

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Adds (non-searchable) docs for Service Catalog beta

### Motivation
DOCS-3541, Discussion with PM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/faq/service_catalog.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+Tags
+: Like all Datadog tags, these help you find, filter, and group whatever they are applied to, in this case service definitions.
+
+Integrations
+: Custom strings for hooking up to integrations such as PagerDuty for identifying on-call for the service.
+
+### Enriching a listed service 
+
+Initially, services that are sending data to Datadog through one of the telemetry products (for example, traces, logs, profiles, infrastructure, or network device monitoring) are listed in the Service Catalog with an `UNDEFINED` label. The `UNDEFINED` label means that no service definition has been associated with that service yet.
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:120]:
```suggestion
Initially, services sending data to Datadog through one of the telemetry products (Traces, Logs, Profiles, Infrastructure, Network Performance, and RUM) are listed with an `UNDEFINED` label, which means that no service definition has been associated with the service yet. 
```

Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:120]:
```suggestion
Initially, services sending data to Datadog through one of the telemetry products (Traces, Logs, Profiles, Infrastructure, Network Performance, and RUM) are listed with an `UNDEFINED` label, which means that no service definition has been associated with the service yet. 
```

File: content/en/tracing/faq/service_catalog.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+: Like all Datadog tags, these help you find, filter, and group whatever they are applied to, in this case service definitions.
+
+Integrations
+: Custom strings for hooking up to integrations such as PagerDuty for identifying on-call for the service.
+
+### Enriching a listed service 
+
+Initially, services that are sending data to Datadog through one of the telemetry products (for example, traces, logs, profiles, infrastructure, or network device monitoring) are listed in the Service Catalog with an `UNDEFINED` label. The `UNDEFINED` label means that no service definition has been associated with that service yet. 
+
+Add service ownership information such as team name, Slack channels, and source code repositories, by using the [Service Definition API][5].
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:122]:
```suggestion
To add service ownership information such as team name, Slack channels, and source code repositories, use the [Service Definition API][5].
```

Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:122]:
```suggestion
To add service ownership information such as team name, Slack channels, and source code repositories, use the [Service Definition API][5].
```

File: content/en/tracing/faq/service_catalog.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+### Registering a new service
+
+You can manage service ownership information even for services that are not emitting any Datadog telemetry. Specify service ownership, on-call info, and custom tags with the [Service Definition API][5], and the information is reflected in the Service Catalog UI.
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:126]:
```suggestion
You can also manage service ownership information for services that do not emit any Datadog telemetry. To register a service, specify the service ownership, on-call information, and custom tags using the [Service Definition API][5]. 
```

Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:126]:
```suggestion
You can also manage service ownership information for services that do not emit any Datadog telemetry. To register a service, specify the service ownership, on-call information, and custom tags using the [Service Definition API][5]. 
```

File: content/en/tracing/faq/service_catalog.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+Initially, services that are sending data to Datadog through one of the telemetry products (for example, traces, logs, profiles, infrastructure, or network device monitoring) are listed in the Service Catalog with an `UNDEFINED` label. The `UNDEFINED` label means that no service definition has been associated with that service yet. 
+
+Add service ownership information such as team name, Slack channels, and source code repositories, by using the [Service Definition API][5].
+
+### Registering a new service
+
+You can manage service ownership information even for services that are not emitting any Datadog telemetry. Specify service ownership, on-call info, and custom tags with the [Service Definition API][5], and the information is reflected in the Service Catalog UI. 
+
+The following YAML shows an example service definition:
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:128]:
```suggestion
See the example service definition YAML:
```

Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:128]:
```suggestion
See the example service definition YAML:
```

File: content/en/tracing/faq/service_catalog.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    provider: github 
+    url: https://github.com/DataDog/shopist/blob/prod/k8s/dd-trace-demo/templates/product-recommendation-lite-deployment.yaml
+docs: 
+  - name: Datadog Doc
+    provider: link
+    url: https://docs.datadoghq.com/tracing/faq/service_catalog/
+tags: []
+```
+
+You can register multiple services with a single YAML file by separating each service definition with a line containing three hyphens: `---`.
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:156]:
```suggestion
To register multiple services in one YAML file, separate each service definition with a line containing three hyphens: `---`.
```

Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:156]:
```suggestion
To register multiple services in one YAML file, separate each service definition with a line containing three hyphens: `---`.
```

File: content/en/tracing/faq/service_definition_api.md [hunk 6]
```diff
@@ -2,16 +2,20 @@
...
+  text: "Datadog Service Catalog"
 ---
 
 ## Services
 
-Generally speaking, a service is an independently deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using other criteria, you can customize your service definition to fit your architectural style in the Datadog Service Catalog.
+Generally speaking, a service is an independently deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using other criteria, you can customize your service definition to fit your architectural style. View your complete service list and gain insights into all services' reliability and security in the [Datadog Service Catalog][2].
 
 
 <div class="alert alert-warning">This feature is in private beta and the endpoints are likely to change.</div>
```
Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:16]: Can we move this beta banner to the top of the page?

Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:16]: Can we move this beta banner to the top of the page?

File: content/en/tracing/faq/service_definition_api.md [hunk 7]
```diff
@@ -2,16 +2,20 @@
 title: Registering Services through the Datadog Service Definition API
 kind: faq
 is_beta: true
+further_reading:
+- link: "/tracing/faq/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
 ---
 
 ## Services
```
Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:11]:
```suggestion
## Overview
```

Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:11]:
```suggestion
## Overview
```

File: content/en/tracing/faq/service_definition_api.md [hunk 8]
```diff
@@ -2,16 +2,20 @@
...
 
-Generally speaking, a service is an independently deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using other criteria, you can customize your service definition to fit your architectural style in the Datadog Service Catalog.
+Generally speaking, a service is an independently deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using other criteria, you can customize your service definition to fit your architectural style. View your complete service list and gain insights into all services' reliability and security in the [Datadog Service Catalog][2].
```
Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:13]:
```suggestion
A service is an independent, deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using additional criteria, customize the service definition that fits your architectural style. View the service list and gain insights into all services' reliability and security in the [Datadog Service Catalog][2].
```

Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:13]:
```suggestion
A service is an independent, deployable unit of software. Datadog [Unified Service Tagging][1] provides a standard way to manage and monitor services consistently across multiple telemetry types, including infrastructure metrics, logs, and traces. If you want to define a service using additional criteria, customize the service definition that fits your architectural style. View the service list and gain insights into all services' reliability and security in the [Datadog Service Catalog][2].
```

File: content/en/tracing/faq/service_definition_api.md [hunk 9]
```diff
@@ -2,16 +2,20 @@
...
 
 
 <div class="alert alert-warning">This feature is in private beta and the endpoints are likely to change.</div>
 
-Before you begin, you need a [Datadog API and app key][2].
+Before you begin, you need a [Datadog API and app key][3].
```
Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:18]:
```suggestion
## Requirements


Before you begin, you need a [Datadog API and app key][3].
```

Comment (alai97) [inline content/en/tracing/faq/service_definition_api.md:18]:
```suggestion
## Requirements

Before you begin, you need a [Datadog API and app key][3].
```

File: content/en/tracing/faq/service_catalog.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+  text: "Service Definition API"
+---
+
+{{< img src="tracing/service_catalog/service_catalog.mp4" video=true alt="Navigating around the Service Catalog" style="width:100%;" >}}
+
+<div class="alert alert-warning">This feature is in private beta and the endpoints are likely to change.</div>
+
+## Overview
+
+Datadog Service Catalog is a centralized place to access important information about all services in your organization. Achieve end-to-end service ownership at scale, get real-time performance insights, detect and address reliability and security risks, and manage application dependencies all in one place. Pivot quickly into team communications tools (Slack), source control (Github), dashboards, and Datadog views that receive and monitor telemetry data for each service.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/faq/service_catalog.md:17]: Took some of these, but some changed the meaning incorrectly.

File: content/en/tracing/faq/service_catalog.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+- Providing engineering leadership with a high-level view of reliability practices across teams and services. 
+- Spotting issues like missing SLOs, monitors, or services without ownership.
+
+## Browse the Service Catalog
+
+On the [Service Catalog page][1], see the list of services in your Datadog organization, either detected from collected data, or defined by someone [registering the service](#registering-a-new-service). To find a particular service, search by its name. To filter the list, select one or more facets. For example, to see detected services that don't yet have a service definition, click the **Ownership Info > Telemetry Only** facet. You might find it helpful to filter by your team name. Or scope the metrics shown to particular environments and clusters to see only matching services in the list.
+
+The list of services in the catalog is sortable service type, service name, as well as many of the other columns, making it quick to find specific or missing details (for example owning team on the Ownership view), and to sort by urgency (for example, number of triggered monitors on the Reliability view).
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/faq/service_catalog.md:35]:
```suggestion
The Service Catalog list is sortable by service type, service name, and many of the other columns. You can find missing ownership by sorting by team in the Ownership view and looking for blanks. Or you can sort by urgency in the Reliability view and see services with the most triggered monitors.
```

File: content/en/tracing/faq/service_catalog.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+Clicking the three-dot menu lets you edit the service definition, if it exists, or link to one if the service hasn't yet been defined. You might have to first [set up an integration with your source code system][2] if you haven't yet.
+
+Sort the table by the Team or On Call columns to quickly spot services where ownership and responsibility have not been specified, or to see which services each team is responsible for.
+
+### Reliability view
+
+The Reliability view brings together key information about the stability of your services. Sort the table by clicking on columns to reveal:
+
+- Which services deployed most recently, or have not deployed for a long time.
+- Which services are reporting the most errors, and are they new issues.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/faq/service_catalog.md:54]:
```suggestion
- Which services are reporting the most errors, and whether they are new issues.
```

File: content/en/tracing/faq/service_catalog.md [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+  text: "Service Definition API"
+---
+
+<div class="alert alert-warning">This feature is in private beta and the endpoints are likely to change.</div>
+
+{{< img src="tracing/service_catalog/service_catalog.mp4" video=true alt="Navigating around the Service Catalog" style="width:100%;" >}}
+
+## Overview
+
+Datadog Service Catalog is a centralized place to access important information about all services in your organization. Achieve end-to-end service ownership at scale, get real-time performance insights, detect and address reliability and security risks, and manage application dependencies all in one place. Access team communications tools such as Slack, source control such as Github, dashboards, and Datadog views that receive and monitor telemetry data for each service.
```
Comment (alai97) [inline content/en/tracing/faq/service_catalog.md:17]:
```suggestion
Datadog Service Catalog is a centralized place to access important information about all services in your organization. Achieve end-to-end service ownership at scale, get real-time performance insights, detect and address reliability and security risks, and manage application dependencies all in one place. Access team communications tools such as Slack, source control such as GitHub, Datadog dashboards, and Datadog views that receive and monitor telemetry data for each service.
```

**Review by alai97 — APPROVED** [review]

**Pull request merged**